crates/oci/src/client.rs

```diff
@@ -34,7 +34,8 @@ const MAX_PARALLEL_PULL: usize = 16;
 pub struct Client {
     /// Global cache for the metadata, Wasm modules, and static assets pulled from OCI registries.
     pub cache: Cache,
-    oci: oci_distribution::Client,
+    /// Underlying OCI client.
+    pub oci: oci_distribution::Client,
```

Comment:
I don't have full context here, but would it make sense to keep this private and expose the relevant customization function through methods on the wrapper `Client` struct instead?

Reply:
Done.